.github/workflows/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
```

Comment:
Please add at least 2 inline comments to your review to fulfil the project requirements🙏

<img width="766" alt="image" src="https://user-images.githubusercontent.com/37116322/220160226-cfdf7bd2-dafc-4350-8935-d7364a80dd22.png">